examples/experimental/dagster-airlift/dagster_airlift/in_airflow/proxying_fn.py

```diff
@@ -21,9 +21,9 @@ def proxying_to_dagster(
     global_vars: Dict[str, Any],
     proxied_state: AirflowProxiedState,
     logger: Optional[logging.Logger] = None,
-    dagster_operator_klass: Type[
-        BaseProxyTaskToDagsterOperator
-    ] = DefaultProxyTaskToDagsterOperator,
+    operator_construction_callback: Optional[
+        Callable[[BaseOperator], BaseProxyTaskToDagsterOperator]
+    ] = None,
```

Comment:
`build_task_fn`

And then we can avoid having to export `build_dagster_task` by instead having `BaseProxyTaskToDagsterOperator` implement a `build_from_task` `@classmethod` that the user can override.